shared/data/constants.js

```diff
@@ -5,12 +5,13 @@ function getConfigFileName () {
     let browserName = browserInfo?.browser?.toLowerCase() || ''
 
     // clamp to known browsers
-    if (!['chrome', 'firefox', 'brave', 'edg'].includes(browserName)) {
+    if (!['chrome', 'firefox', 'brave', 'edg', 'safari'].includes(browserName)) {
         browserName = ''
     } else {
         browserName = '-' + browserName + (chrome?.runtime.getManifest().manifest_version === 3 ? 'mv3' : '')
     }
-    return `https://staticcdn.duckduckgo.com/trackerblocking/config/v2/extension${browserName}-config.json`
+    // return `https://staticcdn.duckduckgo.com/trackerblocking/config/v2/extension${browserName}-config.json`
+    return `https://pyrite-scarce-agenda.glitch.me/staticcdn/trackerblocking/config/v2/extension${browserName}-config.json`
```

Comment:
Lately I've been wondering if we should propose a QWD or similar ability to save this to extension config through the extension dev tools.

Editing list-editor doesn't stick long enough either.